frame/node-permission/src/lib.rs

```diff
@@ -0,0 +1,352 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2019-2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! # Node permission moudule
+//!
+//! This module is used by the `client/service` to retrieve the current
```

Comment:
The docs should be extended and should not mentioning `client/service`.

It should be more like:
```
/// A pallet for managing a set of fixed nodes for a blockchain. 
/// The node set is stored under [`well_known_keys::NODE_ALLOW_LIST`].
/// ...
```